src/data/promise-data.json

```diff
@@ -1,4 +1,35 @@
 [
+  {
+    "name": "MongoDB",
+    "description": "A document-oriented NoSQL database",
+    "logoUrl": "/img/marketplace/mongodb.svg",
+    "available": false,
+    "categories": [
+      "Data"
+    ]
+  },
```

Comment:
So it appears as a "coming soon" on the site, so its about drumming up excitement and capturing interest. Long term we could use it to help prioritise which promises to work on based on how many people click which "coming soon" promise.

Sidenote: If we aren't planning on completing this promise soonish we should probably remove it completely

Reply:
gotcha, makes sense

tbf we have no way of tracking that people actually tried to click on that tile, maybe something to think about.

and yes, it has been there since the launch of the marketplace, many months ago 👀